.husky/pre-push

```diff
@@ -3,6 +3,7 @@
 
 yarn lint:check
```

Comment:
Can this just run `yarn check`? What's the distinction here?

Reply:
I was wondering exactly that.
In a way we have more control (and legibility) over what get executed on push, but we can also forget to add them here!

So I'm ok to update this to use `yarn check` instead.

What do you think @sehyod ?

Reply:
I agree with using `yarn check` in the pre-push hook: it's better not to have to update the code in multiple places every time we want to add a new check